docs/Install.md

```diff
@@ -14,7 +14,40 @@ On Linux the easiest way to install JackTrip is to use the distribution's packag
     sudo apt install jacktrip
     ```
 
-The [GitHub releases page](https://github.com/jacktrip/jacktrip/releases) also includes a binary which should run on most Linux distributions (x64). This build is known to not look well under Wayland.
+### Latest release
+
+If your distribution doesn't include the latest release in their repository, there are different
+alternative options to install from.
+
+=== "Ubuntu"
+
+    Ubuntu users can install from a [PPA repository](https://launchpad.net/~umlaeute/+archive/ubuntu/jacktrip/)
+    maintained by the packager of Ubuntu's official JackTrip packages.
+
+    ```bash
+    sudo add-apt-repository ppa:umlaeute/jacktrip
+    sudo apt update
+    sudo apt install jacktrip
+    ```
+
+=== "Debian"
+
+    Debian stable users can obtain their latest version from the [Debian backports] repository.
+    
+    As an example for Debian Bullseye follow these steps: 
+    
+    ```bash
+    mkdir -p /etc/apt/sources.list.d/
+    echo "deb http://deb.debian.org/debian bullseye-backports main" \
+    | tee /etc/apt/sources.list.d/backports.list
+    apt-get update
+    apt-get install -t bullseye-backports jacktrip
+    ``` 
+
+=== "All"
+
+    The [GitHub releases page](https://github.com/jacktrip/jacktrip/releases) also includes a binary
+    which should run on most Linux distributions (x64). This build is known to not look well under Wayland.
```

Comment:
> This build is known to not look well under Wayland.

Just checking, is this still the case after we updated to Qt 5.15?

Reply:
It's still the case with version 1.7.0. We need to switch to Qt6 for this apparently.